src/controllers/adminController.js

```diff
@@ -0,0 +1,11 @@
+const mongoose = require('mongoose');
```

Comment:
Github actions lint step is pointing out that this import is probably not needed

Reply:
Removed